API_CHANGES.md

```diff
@@ -1,3 +1,24 @@
+# 10.2
+
+## New features
+### Analysis and tracking
+
+* Add `analyzeFilesAndTrack` method to `org.sonarsource.sonarlint.core.rpc.protocol.backend.analysis.AnalysisRpcService` to allow client to submit files for analysis and expect to get a notification with tracked issues.
+  * should be called by the client instead of the deprecated `analyzeFiles` method.
+  * accepts `AnalyzeFilesAndTrackParams` instead of deprecated `AnalyzeFilesParams`. The difference is that the new `AnalyzeFilesAndTrackParams` expects to get one more flag - `shouldFetchServerIssues`.
```

Comment:
Maybe explain when `shouldFetchServerIssues` is supposed to be true (on file open) ?